testsuite/objects/__init__.py

```diff
@@ -35,40 +67,45 @@ class Rule:
     value: str
 
 
-class Value:
-    """Dataclass for specifying a Value in Authorization, can be either constant or value from AuthJson (jsonPath)"""
+@dataclass
+class ValueBase:
+    """
+    Abstract Dataclass for specifying a Value in Authorization,
+    can be either static or reference to value in AuthJson.
+    """
 
-    # pylint: disable=invalid-name
-    def __init__(self, value=None, jsonPath=None) -> None:
-        super().__init__()
-        if not (value is None) ^ (jsonPath is None):
-            raise AttributeError("Exactly one of the `value` and `jsonPath` argument must be specified")
-        self.value = value
-        self.jsonPath = jsonPath
+
+@dataclass
+class ValueStatic(ValueBase):
+    """Dataclass for static Value. Can be anything, eg. string, integer, list, dict..."""
+
+    value: Any
+
+
+@dataclass
+class ValueDynamic(ValueBase):
```

Comment:
I am not convinced creating too many classes especially when AuthJson is always (afaik) used with ValueFrom in structure: `{"valueFrom": {"authJSON": <str>}}` 
Maybe in future if it was used also in other places I would be convinced to make it a separate class. For now I dont think its needed and so it would reduce number of imports in creating Value classes and future classes using the Value.

Renaming of variable and class names is a good idea and I did it.

Reply:
> I am not convinced creating too many classes especially when AuthJson is always (afaik) used with ValueFrom in structure: `{"valueFrom": {"authJSON": <str>}}` Maybe in future if it was used also in other places I would be convinced to make it a separate class. For now I dont think its needed and so it would reduce number of imports in creating Value classes and future classes using the Value.

Agreed